specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/subscriptions.json

```diff
@@ -532,6 +532,178 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Subscription/policies/default": {
+      "put": {
+        "description": "Create or Update Subscription tenant policy for user's tenant.",
+        "operationId": "SubscriptionPolicy_AddUpdatePolicyForTenant",
+        "x-ms-examples": {
+          "TenantPolicy": {
+            "$ref": "./examples/changeTenantPolicy.json"
+          }
+        },
+        "responses": {
+          "200": {
```

Comment:
Using 201 created status is recommended for PUTs

Reply:
Updated